packages/tools/fetch-tool/src/fluidFetchSharePoint.ts

```diff
@@ -56,19 +57,11 @@ export async function resolveWrapper<T>(
         }
         return result;
     } catch (e) {
-        if (e.requestResultError) {
-            const parsedBody = JSON.parse(e.requestResult.data);
-            if (parsedBody.error === "invalid_grant"
-                && parsedBody.suberror === "consent_required"
-                && !forceTokenReauth
-            ) {
-                // Re-auth
-                return resolveWrapper<T>(callback, server, clientConfig, true);
-            }
-            const responseMsg = JSON.stringify(parsedBody.error, undefined, 2);
-            return Promise.reject(`Fail to connect to ODSP server\nError Response:\n${responseMsg}`);
+        if (e.errorType === DriverErrorType.authorizationError && !forceTokenReauth) {
```

Comment:
is this right? `!forceTokenReauth` leads to re-auth?

Reply:
Yes this means that if forceTokenReauth was true then it should have been already done. No need to do again even on error.